staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/internal/modes/decode_test.go

```diff
@@ -200,6 +224,7 @@ func TestDecode(t *testing.T) {
 			into: struct {
 				A int `json:"a"`
 			}{},
+			want:          UncheckedValue{},
```

Comment:
Why is this one unchecked, but down just a bit another unknown field is checking for empty.

Reply:
Any testcase that has `want: UncheckedValue{}` intends only to assert an error in the `assertOnError` field. The want values of these testcases are ignored.